pyproject.toml

```diff
@@ -364,6 +365,32 @@ skip = [                        # Don't check these files/folders
 count = true                    # Display counts of errors
 check-hidden = false            # Don't check hidden files (starting with .)
 
+[tool.mypy]
+namespace_packages = true
+strict_optional = true
+disallow_incomplete_defs = true
+disallow_untyped_defs = true
+strict_bytes = true
+warn_unused_ignores = true
+warn_unused_configs = true
+warn_redundant_casts = true
+ignore_missing_imports = false
+mypy_path = ["src", "packages/tests/src"]
+files = ["src", "tests", "packages/tests/src"]
```

Comment:
Can we just use `packages` instead of `packages/tests/src`? That way we don't need to update this config if we add a new package.